src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -646,15 +721,17 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
         myPrevMembershipData?: CallMembershipData,
         myPrevMembership?: CallMembership,
     ): boolean {
+        if (myPrevMembership && myPrevMembership.getMsUntilExpiry() === undefined) return false;
```

Comment:
only one right?

Reply:
Ah no we cannot do this.
`myPrevMembership && myPrevMembership.getMsUntilExpiry() === undefined` this is different logic: 
`(myPrevMembership?.getMsUntilExpiry() === undefined)` is true if `myPrevMembership = undefined`
while this would be false:
`myPrevMembership && myPrevMembership.getMsUntilExpiry() === undefined`